FAQ.md

```diff
@@ -38,6 +39,16 @@
 - [Where did the name Devito come from?](#where-did-the-name-devito-come-from)
 
 
+## What is Devito?
+Devito isn’t a ready-made library of seismic or wave solvers—it’s a symbolic DSL and code-generation framework for discretizing and solving arbitrary partial differential equations (PDEs). Rather than shipping you a fixed suite of “pre-canned” simulators, Devito, in short, lets you:
```

Comment:
Maybe swap this around so you start with what Devito **is** rather than what it isn't?